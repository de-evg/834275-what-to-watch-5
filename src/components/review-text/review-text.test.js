import React from "react";
import renderer from "react-test-renderer";
import ReviewText from "./review-text";

it(`Should ReviewText render correctly`, () => {
  const tree = renderer
    .create(<ReviewText
      value={`Some value`}
      onTextInput={() => {}}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
