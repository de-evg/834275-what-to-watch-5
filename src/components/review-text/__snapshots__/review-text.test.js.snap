// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should ReviewText render correctly 1`] = `
<textarea
  className="add-review__textarea"
  defaultValue="Some value"
  id="review-text"
  maxLength={400}
  minLength={50}
  name="review-text"
  onInput={[Function]}
  placeholder="Review text"
/>
`;
